### ISSUE TITLE
core sds sequential upload corrupt metadata

### ISSUE THREAD

**Issue by jkasarherou** [open-io/oio-sds#311]
Using core sds
`err = oio_sds_upload(client, &src, &dst)` 

with `OIO_UL_SRC_HOOK_SEQUENTIAL`

```
❯ openio object info 1 x
+-----+----------------------------------------------------------------------------------------+------+----------------------------------+
| Pos | Id                                                                                     | Size | Hash                             |
+-----+----------------------------------------------------------------------------------------+------+----------------------------------+
| 0.0 | http://127.0.0.1:6016/44A700F5089F80A9342698EC32541AE985DF802B54E69441CE07861E3A01D4D6 |   86 | 00000000000000000000000000000000 |
| 0.0 | http://127.0.0.1:6015/A523F205286245B485506B9B02413548097A63D23343A37D6DCD898AEFCBA1DE |   86 | 00000000000000000000000000000000 |
+-----+----------------------------------------------------------------------------------------+------+----------------------------------+

```

The chunks pos and the checksum are wrong

**Comment by jkasarherou** [comment]
positions are fixed in #312

**Comment by jfsmig** [comment]
Check #322 !
